src/services/inplayer.account.service.ts

```diff
@@ -54,50 +54,44 @@ export const getFreshJwtToken = async ({ auth }: { auth: AuthData }) => auth;
 
 export const updateCustomer: UpdateCustomer = async (values) => {
   try {
-    const consents: { [key: string]: string } = {};
-    values.consents?.map((consent: CustomerConsent) => {
-      if (consent.label) {
-        const { customerId, date, newestVersion, needsUpdate, ...rest } = consent;
-        consents[`consents_${consent.name}`] = JSON.stringify(rest);
-      }
-    });
+    const fullName = `${values.firstName} ${values.lastName}`;
```

Comment:
This is kind of a hacky suggestion, but I think it will work to give us slightly better data results - what if we just use two spaces to delimit between first and last name? We can even sanitize the first name and last name inputs here to replace any double spaces with single spaces beforehand. That way we can reliably split on a double space to get back first and last name even when either part has spaces in it itself (i.e. 'Vincent van Gogh' and 'Mary Grace Johnson' will both parse to 'Vincent' / 'van Gogh' and 'Mary Grace' / 'Johnson' appropriately.)  It also will be virtually unnoticeable if there's anywhere else that just renders full name.

Alternatively we could use a comma like 'van Gogh, Vincent'.

Reply:
I think probably the comma solution is a better one. I noticed that we can "split" the fields in the inplayer dashboard. Do we need to handle anything different for that case?

Reply:
Yes, in the InPlayer Dashboard you can split the `fullName` field to **first_name** and **surname**, but this is just metadata and `fullName` remains as it is.
If we decide to go with the comma solution, the existing accounts will see only the first name.

I suggest going with first and last name in the metadata if exists, or as a fallback to whitespace solution

Reply:
OK, then let's use the split metadata fields that inplayer supports (first_name and surname) and make sure to also write to fullname on save.  Then fullname should only be used as a fallback if first_name and surname aren't present.